src/Statistics.jl

```diff
@@ -59,19 +121,38 @@ function mean!(M::Manifold, y, x::AbstractVector, w::AbstractWeights = (n = leng
 end
 
 @doc doc"""
-    median(M, x, weights=Weights(ones(n)); stop_iter=10000)
+    median(M::Manifold, x::AbstractVector; kwargs...)
+    median(M::Manifold, x::AbstractVector, w::AbstractWeights; kwargs...)
 
-computes the Riemannian median of the vector `x`  of `n` points on the
-[`Manifold`](@ref) `M`. This function is nonsmooth (i.e nondifferentiable) and
-uses a cyclic proximal point scheme. Optionally one can provide
-weights $w_i$ for the weighted Riemannian median. The general formula to compute
-the minimizer reads
+Compute the (optionally weighted) Riemannian median of the vector `x` of points on the
+[`Manifold`](@ref) `M`, defined as the point that satisfies the minimizer
 ````math
-\argmin_{y\in\mathcal M}\sum_{i=1}^n w_i\mathrm{d}_{\mathcal M}(y,x_i),
+\argmin_{y\in\mathcal M} \frac{1}{\sum w} \sum_{i=1}^n w_i\mathrm{d}_{\mathcal M}(y,x_i),
```

Comment:
see note above, I think this should read `\sum_i w_i` instead of just ` \sum w`.